src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/components/enterprise-alert/index.tsx

```diff
@@ -82,14 +98,14 @@ export function EnterpriseAlert({
 }
 
 type DocsEnterpriseAlertProps = Omit<EnterpriseAlertProps, 'productSlug'> & {
-	product?: ProductSlug
+	product?: ProductSlug | 'hcp-vault-secrets' | 'hcp-vault-radar'
 }
 
 export function DocsEnterpriseAlert(props: DocsEnterpriseAlertProps) {
 	const currentProduct = useCurrentProduct()
 	return (
 		<EnterpriseAlert
-			productSlug={props.product || currentProduct.slug}
+			product={props.product || currentProduct.slug}
```

Comment:
Personally, I like the nullish coalescing operator in case the left-hand value is an empty string, but this is not a blocker
```suggestion
			product={props.product ?? currentProduct.slug}
```